specs/icmp.rflx

```diff
@@ -0,0 +1,27 @@
+package ICMP is
+
+    type Tag is (Echo_Reply => 0, Echo_Request => 8)
+        with Size => 8;
+
+    type Code is range 0 .. 0
+        with Size => 8;
```

Comment:
To be consistent with our other spec files the `with` clause should be on the same line.

Reply:
fixed and squashed into 6d14bb9